docs/transaction.rst

```diff
@@ -0,0 +1,57 @@
+Transaction Operations
+======================
+
+Transaction operations are supported using context managers. The DynamoDB API has limits on the number of items in
+each request, but PynamoDB doesn't currently handle grouping or paginating, so this is something you must handle on your
+own.
+
+.. note::
+
+    An ordered array of up to 25 `TransactWriteItem` objects, each of which contains a `ConditionCheck`, `Put`, `Update`, or `Delete` object. These can operate on items in different tables, but the tables must reside in the same AWS account and Region, and no two of them can operate on the same item.
+
+Suppose you have defined a BankStatement model, like in the example below.
+
+.. code-block:: python
+
+    from pynamodb.models import Model
+    from pynamodb.attributes import BooleanAttribute, NumberAttribute, UnicodeAttribute
+
+    class BankStatement(Model):
+        class Meta:
+            table_name = 'BankStatement'
+
+    user_id = UnicodeAttribute(hash_key=True)
+    account_balance = NumberAttribute(default=0)
+    is_active = BooleanAttribute()
+
+
+Transact Writes
+^^^^^^^^^^^^^^^
+
+Here's an example of using a context manager for a `TransactWrite` operation:
+
+.. code-block:: python
+
```

Comment:
It would be helpful to include the import here, i.e

```
from pynamodb.connection.transactions import TransactWrite
```

As I write this, I'm also realizing that this API expects folks to import out of `.connection`, which we would usually reserve for lower-level operations. What do you think about bumping the package up one level for the 4.0 release? i.e `from pynamodb.transactions import ...`?

Reply:
sure, i can move it :0

Reply:
https://github.com/pynamodb/PynamoDB/pull/667